frontend/app/utils/date-utils.ts

```diff
@@ -0,0 +1,55 @@
+import { useMemo } from 'react';
+
+import { padWithZero } from './string-utils';
```

Comment:
To be consistent with the rest of the application, this should be an aliased (`~/`) import.